### PR TITLE
QR Code Scanner

### PR DESCRIPTION
**Pull request by alerobertson**
Note: The scanner usually requires HTTPS on a few mobile browsers, so it may not work locally

### CODE REVIEW

**Comment by alerobertson** [comment]
The scanner only accepts you on the following conditions:

- The ID on the QR code leads to a valid applicant
- The applicant in question is either "accepted", "attending", "not-attending" OR they have been signed-in
- The ID has not been signed in to the event in question

**Comment by alerobertson** [comment]
The scanner cannot scan the same code twice in a row. It's something with the packaged component, but you can refresh the page to get by it.

File: src/components/Scanner.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 <script>
 import {QrcodeStream} from 'vue-qrcode-reader';
-import axios from 'axios';
```
Comment (fpunny) [inline src/components/Scanner.vue]: Im proud of u. Feel free to talk to me or my son ever again

**Review by lyninx — APPROVED** [review]
tested locally! looks good! 💯

File: src/components/Scanner.vue [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      response: 'Ready',
+      success: true,
+      response_color: '#FFFFFF',
+    };
+  },
+  methods: {
+    handler({target}) {
+      this[target.name] = target.value;
+    },
+    async onInit() {
```
Comment (fpunny) [inline src/components/Scanner.vue:33]: Kinda iffy here but it works xD

**Review by fpunny — APPROVED** [review]

**Pull request merged**